docs/source-pytorch/advanced/third_party/hivemind.rst

```diff
@@ -0,0 +1,114 @@
+:orphan:
+
+################################################################
+Hivemind - training on unreliable mixed GPUs across the internet
+################################################################
+
+Collaborative Training tries to solve the need for top-tier multi-GPU servers by allowing you to train across unreliable machines,
+such as local machines or even preemptible cloud compute across the internet.
+
+Under the hood, we use `Hivemind <https://github.com/learning-at-home/hivemind>`_ which provides de-centralized training across the internet.
+
+.. warning::  This is an :ref:`experimental <versioning:Experimental API>` feature.
+
+
+To use Collaborative Training, you need to first this extension.
+
+.. code-block:: bash
+
+    pip install lightning-Hivemind
```

Comment:
why is the package capitalized? Shouldn't it be `pip install lightning-hivemind`?

Reply:
```suggestion
    pip install lightning-hivemind
```